tests/composition/test_composition.py

```diff
@@ -3108,12 +3108,12 @@ def test_some_inputs_not_specified(self):
         sched = Scheduler(composition=comp)
         comp.run(inputs=inputs, scheduler=sched)[0]
 
-        assert np.allclose(A.get_output_values(comp), [[2.0, 4.0], [6.0, 8.0]])
-        assert np.allclose(B.get_output_values(comp), [[3., 6., 9.]])
-        assert np.allclose(C.get_output_values(comp), [[0.]])
-        assert np.allclose(D.get_output_values(comp), [[4.]])
+        np.testing.assert_allclose(A.get_output_values(comp), [[2.0, 4.0], [6.0, 8.0]])
+        np.testing.assert_allclose(B.get_output_values(comp), [[3., 6., 9.]])
+        np.testing.assert_allclose(C.get_output_values(comp), [[0.]])
+        np.testing.assert_allclose(D.get_output_values(comp), [[4.]])
         for i,j in zip(comp.results[0], [[2., 4.], [6., 8.], [3., 6., 9.],[0.], [4.]]):
```

Comment:
Can  the zip iteration be replaced by a single `np.testing.assert_allclose`?

Reply:
This one can't for the same ragged array reason